mxcube3/routes/qutils.py

```diff
@@ -344,12 +349,17 @@ def set_dc_params(model, entry, task_data):
 
     acq.acquisition_parameters.set_from_dict(params)
     acq.path_template.set_from_dict(params)
+    acq.path_template.base_prefix = params['prefix']
 
     full_path = os.path.join(mxcube.session.get_base_image_directory(),
                              params.get('path', 'dummy_path'))
 
     acq.path_template.directory = full_path
 
+    process_path = os.path.join(mxcube.session.get_base_process_directory(),
+                                params.get('path', 'dummy_path'))
```

Comment:
What is 'dummy_path' used for ? Is it to prevent a KeyError exception, in this case 'dummy_path' is not much better ?